src/resonanceReconstruction/rmatrix/Channel/src/sommerfeldParameter.hpp

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
Since you moved this file, you need to re-run `metaconfigure/generate.py`. I'm getting error that the file can't be found.

This will be a nice thing to go away when we have the newer build system.

Reply:
I'm not done yet. I'm going over each class separately and compiling each test separately. No need to commit the same file over and over simply because I rename files.

Reply:
Fair enough. I was just trying to compile to see if the tests pass. Let me know when I should try again.

Reply:
It's gonna take a while. I hope to be done today.